gems/smithy/lib/smithy/views/client/endpoint_parameter.rb

```diff
@@ -25,15 +25,25 @@ def docstrings
         end
 
         def documentation_type
-          case @data['type']
-          when 'stringArray' then 'Array<String>'
+          case @data['type'].downcase
+          when 'stringarray' then 'Array<String>'
```

Comment:
I'd prefer not downcasing if we know the casing (these are straight from the endpoints specification and not user input right?)